pegasus/sites.v3/code.org/views/video_with_fallback.haml

```diff
@@ -1,5 +1,8 @@
+- download_path ||= nil
+- poster_path ||= "//i.ytimg.com/vi/#{video_code}/mqdefault.jpg"
```

Comment:
I do wonder if schools that block YouTube also block ytimg 🤔 If ytimg is blocked, users will just see the black rectangles right? They would still be able to play the video?

Reply:
fair fair. Sounds good!